results/proto/api.proto

```diff
@@ -3,12 +3,15 @@ syntax = "proto3";
 package tekton;
 
 import "taskrun.proto";
+import "google/protobuf/empty.proto";
 
 option go_package = "./proto";
 
 service Results {
+  rpc CreateTaskRun (CreateTaskRunRequest) returns (TaskRun) {}
   rpc UpdateTaskRun (UpdateTaskRunRequest) returns (TaskRun) {}
   rpc GetTaskRun    (GetTaskRunRequest)    returns (TaskRun) {}
+  rpc DeleteTaskRun (DeleteTaskRunRequest) returns (google.protobuf.Empty) {}  
```

Comment:
We should avoid adding new methods, since this PR is only focusing on Create.

Reply:
done